### PR TITLE
Add tabs for Integrations Phase 2

### PR DESCRIPTION
**Pull request by fhlavac**
Part of [RHCLOUD-28950](https://issues.redhat.com/browse/RHCLOUD-28950)

Merge after https://github.com/RedHatInsights/notifications-frontend/pull/485

image

### CODE REVIEW

File: src/components/TabNavigation.js [hunk 1]
```diff
@@ -50,14 +52,34 @@ const TabNavigation = () => {
...
       />
-      {enableIntegrations && (
-        <Tab
-          eventKey={INTEGRATIONS}
-          title={
-            <TabTitleText>{intl.formatMessage({ id: 'sources.integrations', defaultMessage: 'Integrations' })}</TabTitleText>
-          }
-        />
-      )}
+      {enableIntegrations &&
```
Comment (karelhala) [inline src/components/TabNavigation.js:55]: Can we have a fallback to phase 1? If `enableIntegrations` is correct we'll show whole integrations table, if the `enableBreakdown` is true we'll show the list of tabs.

File: src/pages/Sources.js [hunk 2]
```diff
@@ -185,8 +186,8 @@ const SourcesPage = () => {
   const setSelectedType = (selectedType) => stateDispatch({ type: 'setSelectedType', selectedType });
 
   const mainContent =
-    activeCategory === INTEGRATIONS && enableIntegrations ? (
-      <AsyncComponent appName="notifications" module="./IntegrationsTable" />
+    [INTEGRATIONS, COMMUNICATIONS, REPORTING, WEBHOOKS].includes(activeCategory) && enableIntegrations ? (
+      <AsyncComponent appName="notifications" module="./IntegrationsTable" activeCategory={activeCategory} />
```
Comment (karelhala) [inline src/pages/Sources.js:190]: Same as above, we probably want to fallback to previous implementation if `platform.sources.breakdown` is disabled.

**Review by karelhala — COMMENTED** [review]
Looking good! Just two minor things.

File: src/pages/Sources.js (hunk 2 above)
Comment (fhlavac) [reply src/pages/Sources.js:190]: Tested again and I hope it works this way now

File: src/components/TabNavigation.js (hunk 1 above)
Comment (fhlavac) [reply src/components/TabNavigation.js]: Thank you, this needed an update, done.

**Review by karelhala — APPROVED** [review]

**Pull request merged**